cypress/e2e/settings/marc-authority/edit-all-editable-fields-of-local-authority.cy.js

```diff
@@ -0,0 +1,136 @@
+        hridStartsWith: '1',
+        baseUrl: '',
+        source: 'Local',
+        isActive: true,
+        createdByAdmin: `${date} by ADMINISTRATOR, Diku_admin`,
+      };
+
+      const fieldsToUpdate = {
+        name: `C436844 Local source Updated by user (all fields)${getRandomPostfix()}`,
+        prefix: getRandomLetters(7),
+        hridStartsWith: '12000',
+        baseUrl: `https://testing/field/baseurl/positivetest3${getRandomLetters(4)}/`,
+      };
+      const fieldTitles = {
+        NAME: 'Name',
+        PREFIX: 'Prefix',
+        HRID_STARTS_WITH: 'HRID starts with',
+        BASEURL: 'Base URL',
+      };
+      let user;
+
+      before('Create users, data', () => {
+        cy.getAdminToken();
+        cy.createTempUser([Permissions.uiSettingsManageAuthorityFiles.gui])
+          .then((userProperties) => {
+            user = userProperties;
+          })
+          .then(() => {
+            cy.createAuthoritySourceFileUsingAPI(
+              localAuthFile.prefix,
+              localAuthFile.hridStartsWith,
+              localAuthFile.name,
+              localAuthFile.isActive,
+            ).then((sourceId) => {
+              localAuthFile.id = sourceId;
+            });
+          });
+      });
+
+      after('Delete users, data', () => {
+        cy.getAdminToken();
+        Users.deleteViaApi(user.userId);
+        cy.deleteAuthoritySourceFileViaAPI(localAuthFile.id, true);
+      });
+
+      it(
+        'C436844 Edit all editable fields of Local "Authority file" which does not have assigned "MARC authority" records (spitfire)',
+        { tags: ['criticalPath', 'spitfire'] },
+        () => {
+          // 1 Go to "Settings" app >> "MARC authority" >> "Manage authority files"
+          cy.login(user.username, user.password, {
+            path: TopMenu.settingsAuthorityFilesPath,
+            waiter: ManageAuthorityFiles.waitLoading,
+          });
+          ManageAuthorityFiles.checkManageAuthorityFilesPaneExists();
+          ManageAuthorityFiles.checkAuthorityFilesTableExists();
+          ManageAuthorityFiles.checkSourceFileExists(
+            localAuthFile.name,
+            localAuthFile.prefix,
+            localAuthFile.hridStartsWith,
+            localAuthFile.baseUrl,
+            localAuthFile.isActive,
+            localAuthFile.createdByAdmin,
+            true,
+          );
+
+          // 2 Click on the "Edit" (pencil) icon of "Local" authority file which has assigned "MARC authority" records
+          ManageAuthorityFiles.clickEditButton(localAuthFile.name);
+          ManageAuthorityFiles.checkRowEditableInEditMode(
+            localAuthFile.name,
+            localAuthFile.prefix,
+            localAuthFile.hridStartsWith,
+            localAuthFile.baseUrl,
+            localAuthFile.source,
+            localAuthFile.createdByAdmin,
+          );
+          ManageAuthorityFiles.checkNewButtonEnabled(false);
+
+          // 3 Update all editable fields with unique valid value, ex:
```

Comment:
Do you need this commented code?

Reply:
These lines are a description of the step, but maybe you're right, it's redundant. Deleted them.